File: Leetcode-Easy/9-PalindromeNumber/Solution.cpp
class Solution {
public:
    bool isPalindrome(int x) {
        if(x<0) return false;
        int n = x;
        unsigned int sum=0;
        while(n!=0){
            sum = sum*10 + n%10;
            n /= 10;
        }
        return sum == x;
    }
};

File: Leetcode-Easy/9-PalindromeNumber/Solution.py
class Solution(object):
    def isPalindrome(self, x):
        if (x < 0):
            return False;
        dup = x;
        num = 0;
        while (dup > 0):
            d = dup % 10;
            num = num*10 + d;
            dup = dup//10;
        return num==x;